aries-site/src/examples/templates/content-layouts/ContentDrivenLayout.js

```diff
@@ -0,0 +1,111 @@
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Notification,
+  Paragraph,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { ModalBody, ModalDialog, ModalFooter } from '../ModalDialog';
+
+const path = '/products/data/47287498729/collections/KCHDvfcByKvvjymNheg';
+const parts = path.split('/').map(part => `/${part}`);
+const matchPath = [
+  ...parts.map(part => ({
+    regexp: new RegExp(part),
+    message: `Location paths do not match. Expecting ${part}.`,
+    status: 'error',
+  })),
+  {
+    regexp: new RegExp(path),
+    message: 'Location paths do not match.',
+    status: 'error',
+  },
+  function validate(value) {
+    if (value !== path) {
+      return {
+        message: `Location paths do not match. 
+Expected: ${path}
+Received: ${value}`,
+        status: 'error',
+      };
+    }
+    return undefined;
+  },
+];
+
+export const ContentDrivenLayout = ({ containerRef }) => {
+  // containerRef is for demonstration purposes on this site. Most
+  // implementations should likely remove.
+  const [showModal, setShowModal] = useState(true);
+  const [value, setValue] = useState({ deletionPath: '' });
+  const onClose = () => setShowModal(false);
+  const onChange = nextValue => {
+    setValue(nextValue);
+  };
+
+  return (
+    <Box align="center" justify="center" fill>
+      <Button
+        primary
+        label="Display Modal"
+        onClick={() => setShowModal(true)}
+      />
+      {showModal ? (
+        <ModalDialog
+          title="Delete Data"
+          onClose={onClose}
+          onEsc={onClose}
+          target={containerRef && containerRef.current}
+        >
+          <Form value={value} onChange={onChange} validate="blur">
+            <Box gap="medium">
+              <ModalBody gap="small">
+                <Notification
+                  status="critical"
+                  message="This action cannot be undone."
+                />
+                <>
+                  <Paragraph margin="none">
+                    This will permanently delete all data, including nested
+                    data, located at:
+                  </Paragraph>
+                  <Text weight="bold">{path}</Text>
+                </>
+                <FormField
+                  htmlFor="deletionPath"
+                  name="deletionPath"
+                  label={
+                    <>
+                      <Text>
+                        To confirm, please type: <br />
+                        {path}
+                      </Text>
+                    </>
+                  }
+                  validate={matchPath}
+                >
+                  <TextInput id="deletionPath" name="deletionPath" />
+                </FormField>
+              </ModalBody>
+              <ModalFooter>
+                <></>
+                <Box direction="row" gap="xsmall">
+                  <Button label="Cancel" onClick={onClose} />
+                  <Button primary label="Delete" onClick={() => {}} />
```

Comment:
should we have `Delete Data` for the label? Similar to the delete cluster example in Forms page

Reply:
I think @CitizenFox is actively working on that guidance. I think the direction we are going is to treat the noun (e.g. "data") as optional. If the content/flow is focussed and clear on a specific object, then it is not necessary. If there is ambiguity, add the noun.

@CitizenFox - Is my understanding ☝️  correct? Here's the example for reference:
![Screen Shot 2022-04-22 at 12 16 52 PM](https://user-images.githubusercontent.com/1756948/164771834-ea01587c-77c1-478e-9b53-aecffb3f4611.png)

Reply:
got it be good to clarify

Reply:
You are 100% correct, @halocline!